mathics_django/doc/django_doc.py

```diff
@@ -65,58 +70,27 @@ def get_title_html(self):
 
 
 class DjangoDocumentation(Documentation, DjangoDocElement):
```

Comment:
In the current mathics-core master branch, the `Documentation` class just has attributes related to the structure of the documentation, but not the part of loading `DocPart`s from modules, which is included in `MathicsMainDocumentation`. This class should subclass this other class.

Reply:
This has been chained in master now.  MathicsMainDocumentation was intended for testing only. LaTeX and HTML documentation are subclasses of Documentation.